src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1591,6 +1591,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("DEBUG_FLOAT_ARRAY", 1.0f);
 		configure_stream_local("DEBUG_VECT", 1.0f);
 		configure_stream_local("DISTANCE_SENSOR", 0.5f);
+		configure_stream_local("ESC_STATUS", 50.0f);
```

Comment:
This needs to be much lower or not included at all in the normal mode. These stream settings need to be usable on SiK radios.

Reply:
@dagar is there a place where I can read about which mode is which and when is it used? I wasn't aware that this was the case here but in that case I will make it much smaller. Perhaps 1Hz is ok?